node/task.ts

```diff
@@ -183,8 +184,12 @@ export function setVariable(name: string, val: string, secret: boolean = false):
     // store the metadata
     im._knownVariableMap[key] = <im._KnownVariableInfo>{ name: name, secret: secret };
 
-    // write the command
-    command('task.setvariable', { 'variable': name || '', 'issecret': (secret || false).toString() }, varValue);
+    // write the command - either with isOutput=true included or not
+    if (isOutput) {
```

Comment:
Nit: Rather than branching into an `if` on `isOutput`, I'd prefer to inline the expression like we do for `isSecret` (for consistency more than anything else) - so `command('task.setvariable', { 'variable': name || '', isoutput: (isOutput || false).toString(), 'issecret': (secret || false).toString() }, varValue);

Reply:
Done.